pages/test_analytics/test_suites.md.erb

```diff
@@ -24,6 +24,18 @@ Organizations typically choose their main production branch as their default, al
 
 To change your default branch, go to suite settings. You can also filter Test Analytics views by any branch by typing its name into the branch query parameter in the Test Analytics URL.
 
+## Tracking down flaky tests
```

Comment:
_Tracking down_ is a bit colloquial and doubles up "Tracking" in the navigation. How about using an alternative?

```suggestion
## Detecting flaky tests
```

I also wondered about finding, discovering, or uncovering, but I think "detecting" works best here.